### PR TITLE
OSSM-4592: Fix indentation in ossm-config-operator-infrastructure-node

### PR DESCRIPTION
**Pull request by rh-tokeefe**
…e.adoc

Version(s): 4.10+

Issue: https://issues.redhat.com/browse/OSSM-4592

Link to docs preview: https://63368--docspreview.netlify.app/openshift-enterprise/latest/service_mesh/v2x/installing-ossm#ossm-config-operator-infrastructure-node_installing-ossm

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/ossm-config-operator-infrastructure-node.adoc [hunk 1]
```diff
@@ -18,29 +18,36 @@ If the operator will run on a worker node, skip this task.
 
 .Procedure
 
+. List the operators installed in the namespace by running the following syntax at the CLI:
```
Comment (luksa) [inline modules/ossm-config-operator-infrastructure-node.adoc:21]: Do we use "syntax"? Don't we usually just say "command"?

File: modules/ossm-config-operator-infrastructure-node.adoc [hunk 2]
```diff
@@ -18,29 +18,36 @@ If the operator will run on a worker node, skip this task.
...
+----
+
 . Edit the {SMProductShortName} Operator `Subscription` resource to specify where the operator should run:
 +
 [source,terminal]
 ----
 $ oc -n openshift-operators edit subscription <name> <1>
 ----
-<1> `<name>` represents the name of the `Subscription` resource.
+<1> `<name>` represents the name of the `Subscription` resource. The default name of the Subscription resource is `servicemesoperator`.
```
Comment (luksa) [inline modules/ossm-config-operator-infrastructure-node.adoc:34]:
```suggestion
<1> `<name>` represents the name of the `Subscription` resource. The default name of the Subscription resource is `servicemeshoperator`.
```

File: modules/ossm-config-operator-infrastructure-node.adoc (hunk 1 above)
Comment (rh-tokeefe) [reply modules/ossm-config-operator-infrastructure-node.adoc:21]: old habits die hard, as they say. Thanks for noting this!

File: modules/ossm-config-operator-infrastructure-node.adoc (hunk 2 above)
Comment (rh-tokeefe) [reply modules/ossm-config-operator-infrastructure-node.adoc:34]: Ugh! Thanks for catching this @luksa!

**Review by luksa — APPROVED** [review]
LGTM

File: modules/ossm-config-operator-infrastructure-node.adoc (hunk 1 above)
Comment (rh-tokeefe) [reply modules/ossm-config-operator-infrastructure-node.adoc]: done

File: modules/ossm-config-operator-infrastructure-node.adoc (hunk 2 above)
Comment (rh-tokeefe) [reply modules/ossm-config-operator-infrastructure-node.adoc]: done

**Review by pbajjuri20 — APPROVED** [review]

File: modules/ossm-config-operator-infrastructure-node.adoc [hunk 3]
```diff
@@ -18,29 +18,36 @@ If the operator will run on a worker node, skip this task.
 
 .Procedure
 
+. List the operators installed in the namespace by running the following command at the CLI:
```
Comment (ekristova) [inline modules/ossm-config-operator-infrastructure-node.adoc:21]: Just a suggestion,  the end of the sentence could be omitted completely:
```suggestion
. List the operators installed in the namespace:
```

File: modules/ossm-config-operator-infrastructure-node.adoc [hunk 4]
```diff
@@ -18,29 +18,36 @@ If the operator will run on a worker node, skip this task.
...
+----
+
 . Edit the {SMProductShortName} Operator `Subscription` resource to specify where the operator should run:
 +
 [source,terminal]
 ----
 $ oc -n openshift-operators edit subscription <name> <1>
 ----
-<1> `<name>` represents the name of the `Subscription` resource.
+<1> `<name>` represents the name of the `Subscription` resource. The default name of the Subscription resource is `servicemeshoperator`.
```
Comment (ekristova) [inline modules/ossm-config-operator-infrastructure-node.adoc:34]:
```suggestion
<1> `<name>` represents the name of the `Subscription` resource. The default name of the `Subscription` resource is `servicemeshoperator`.
```

File: modules/ossm-config-operator-infrastructure-node.adoc (hunk 3 above)
Comment (rh-tokeefe) [reply modules/ossm-config-operator-infrastructure-node.adoc:21]: done

File: modules/ossm-config-operator-infrastructure-node.adoc (hunk 4 above)
Comment (rh-tokeefe) [reply modules/ossm-config-operator-infrastructure-node.adoc:34]: done

**Comment by rh-tokeefe** [comment]
@ekristova the PR was rebased and the changes were made. Confirmed changes in preview build. Please merge when you have a moment. Thanks!

**Review by eromanova97 — APPROVED** [review]

**Comment by ekristova** [comment]
Hello @rh-tokeefe I checked and approved the PR as the changes look good to me, however, I am not a part of merge team, only review team, so I cannot merge the task for you. Please proceed with the merge process as needed.
Thank you!

**Review by gwynnemonahan — APPROVED** [review]
lgtm

**Pull request merged**